build/Makefile.generate_proto

```diff
@@ -0,0 +1,84 @@
+# Copyright 2022 The SODA Authors.
```

Comment:
IMHO, The script generates grpc boilerplate code. It can be moved out from this PR as it is not used

Reply:
yep thought of doing later, any how  removed it